### PR TITLE
tests: add tests for NumberInput

### PR DESCRIPTION
**Pull request by maratori**

### CODE REVIEW

File: tests/test_0001_html.py [hunk 1]
```diff
@@ -461,10 +461,10 @@ def test_not_implemented_input_types(self, tp):
         assert type(node) is Node
 
     def test_input_type_number(self):
-        node = HTML('<input type="number" value="123.5"/>')[0]
```
Comment (fscherf) [inline tests/test_0001_html.py:464]: @maratori why is this changed?

Comment (maratori) [reply tests/test_0001_html.py]: W/o any specific purpose. Reverted.

**Comment by fscherf** [comment]
@maratori Great test! Thanks :)
